eclipse-collections-code-generator/src/main/resources/test/map/immutableObjectPrimitiveMapFactoryImplTest.stg

```diff
@@ -1,52 +0,0 @@
-import "copyright.stg"
-import "primitiveHashCode.stg"
-import "primitiveLiteral.stg"
-
-isTest() ::= "true"
-
-targetPath() ::= "org/eclipse/collections/impl/map/primitive"
-
-fileName(primitive) ::= "ImmutableObject<primitive.name>MapFactoryImplTest"
-
-skipBoolean() ::= "true"
-
-class(primitive) ::= <<
-<body(primitive.type, primitive.name)>
->>
-
-body(type, name) ::= <<
-<copyright()>
-
-package org.eclipse.collections.impl.map.primitive;
-
-import org.eclipse.collections.impl.factory.primitive.Object<name>Maps;
-import org.eclipse.collections.impl.map.mutable.primitive.Object<name>HashMap;
-import org.junit.Assert;
-import org.junit.Test;
-
-/**
- * This file was automatically generated from template file immutableObjectPrimitiveMapFactoryImplTest.stg.
- */
-public class ImmutableObject<name>MapFactoryImplTest
```

Comment:
Why delete these tests?

Reply:
I think they are just renamed? Is there any way to get the diff to look a little simpler in the review?

Reply:
The tests for these factories were non-non-existent and lacked coverage. If you review the whole PR instead of 1 commit at a time it'll be easier. Splitting the PR will be difficult. For transparency: I have moved all the tests from previous test classes and added more tests such that the factories are now covered.